scipy/stats/stats.py

```diff
@@ -3339,9 +3339,45 @@ def pointbiserialr(x, y):
     pvalue : float
         2-tailed p-value
 
+    Notes
+    -----
+    For testing the hypothesis that the point-biserial correlation coefficient
+    is significantly different from zero, a t-test with n-2 degrees of freedom
+    should be used. Pearsonr is equivalent.
+
+    The value of the point-biserial correlation can be calculated from:
+
+    .. math::
+
+        r_{pb} = \frac{\overline{Y_{1}} - \overline{Y_{0}}}{\overline{s_{y}}}\sqrt{\frac{N_{1} N_{2}}{N (N - 1))}}
+
+    Where ``:math:Y_{0}`` and ``:math:Y_{1}`` are means of the metric
+    observations coded 0 and 1 respectively; ``:math:N_{0}`` and ``:math:N_{1}``
+    are number of observations coded 0 and 1 respectively; ``:math:N`` is the
+    total number of observations and ``:math:\overline{s_{y}}`` is the standard
```

Comment:
I think you can drop the ``\overline`` from ``s_y``, it doesn't really serve a purpose. For Y0/Y1 it does (it's the symbol for "mean").